WordPress/src/main/java/org/wordpress/android/models/Account.java

```diff
@@ -37,6 +40,53 @@ public void onErrorResponse(VolleyError volleyError) {
         WordPress.getRestClientUtilsV1_1().get("me", listener, errorListener);
     }
 
+    public void fetchAccountSettings(final ApiHelper.GenericCallback callback) {
+        com.wordpress.rest.RestRequest.Listener listener = new RestRequest.Listener() {
+            @Override
+            public void onResponse(JSONObject jsonObject) {
+                if (jsonObject != null) {
+                    updateAccountSettingsFromRestResponse(jsonObject);
+                    save();
+                    if (callback != null) {
```

Comment:
Using a callback is a good approach, but over the past several months we've been leaning towards using `EventBus` instead. You can see a number of examples in the Reader code (and plenty of callback examples, too, that were added before we used `EventBus`).